src/tzstats/tzstats_reward_provider_helper.py

```diff
@@ -288,10 +276,17 @@ def get_cycle_total_stake(self, cycle):
             # This means something went wrong.
             raise ApiProviderException("GET {} {}".format(uri, resp.status_code))
 
-        staking_supply = resp.json()["staking_supply"]
+        resp_json = resp.json()
+        staking_supply = resp_json["snapshot_cycle"]["staking_supply"]
         return staking_supply
 
-    def __fetch_current_balance(self, address_list):
+    def get_current_cycle(self):
+        uri = self.api + tip
+        resp = requests.get(uri, timeout=5)
+        root = resp.json()
+        return root["cycle"]
+
+    def fetch_current_balance(self, address_list):
```

Comment:
are we planning to use this externally? Am I missing where this is happening?

Reply:
No I don't think so. I was just renaming it for better readability and later we can also unit test those functions since we can then access them externally. Currently the usage of the private prefixes is not used in the entire code base thus I would like to avoid using `__` for now.